settings.gradle

```diff
@@ -48,6 +48,10 @@ include "beam-runners-gearpump"
 project(":beam-runners-gearpump").dir = file("runners/gearpump")
 include "beam-runners-google-cloud-dataflow-java"
 project(":beam-runners-google-cloud-dataflow-java").dir = file("runners/google-cloud-dataflow-java")
+// These 2 projects will not be published to Maven so they don't get a special
+// dashed name.
+include ":runners:google-cloud-dataflow-java:examples:preCommit"
```

Comment:
I don't think you intended to add the ":preCommit" on the end.

Reply:
fixed.